--- src/Entity/PathautoPattern.php
+++ src/Entity/PathautoPattern.php
@@ -429,7 +429,7 @@
         return $this->getContextFromProperty($property_path, $contexts[$base]);
       }
       // @todo improve this exception message.
-      throw new ContextNotFoundException("The requested context was not found in the supplied array of contexts.");
+      throw new \Drupal\Component\Plugin\Exception\ContextException("The requested context was not found in the supplied array of contexts.");
     }
   }
